docs/how-does-it-work/wtf-is-metagame.md

```diff
@@ -1,108 +1,173 @@
 ---
 title: 🤨 WTF is MetaGame?
-image: https://i.imgur.com/N7eJluF.png
+image: https://substackcdn.com/image/fetch/f_auto,q_auto:good,fl_progressive:steep/https%3A%2F%2Fbucketeer-e05bbc84-baa3-437e-9518-adb32be77984.s3.amazonaws.com%2Fpublic%2Fimages%2F43b8522a-6906-424e-ac28-a258e971f8dc_1191x621.png
 ---
 
-## Intro
-ETH is money, Ethereum is a State, dApps are public utilities; we are global citizens.
+[](https://substackcdn.com/image/fetch/f_auto,q_auto:good,fl_progressive:steep/https%3A%2F%2Fbucketeer-e05bbc84-baa3-437e-9518-adb32be77984.s3.amazonaws.com%2Fpublic%2Fimages%2F43b8522a-6906-424e-ac28-a258e971f8dc_1191x621.png)
 
-What drew us together is the idea that Ethereum is a critical piece of technology to be used for building better socioeconomic systems & infrastructure.
+## 👉 Intro[​](https://wiki.metagame.wtf/how-does-it-work/wtf-is-metagame#intro)
 
-On our way here, we’ve also realized it’s not really about Ethereum but about us, the community. It’s not Ethereum that made us global citizens, it’s the realization that the nation-states we’re born under define us about as much as the socks we wear.
+> _**“Metagame**, or **game about the game**, is any approach to a game that transcends or operates outside of the prescribed rules of the game, uses external factors to affect the game, or goes beyond the supposed limits or environment set by the game.”_
 
-We are here because we believe there is an opportunity to do things better, and we’ll be doing that no matter what technological or non-technological means it takes. We are the optimistic new kids on the block. Let’s play.
+What drew us together is the idea that Ethereum is a critical piece of technology for building better socioeconomic systems - a game-changing technology.
 
-> _**“Metagame**, or **game about the game**, is any approach to a game that transcends or operates outside of the prescribed rules of the game, uses external factors to affect the game, or goes beyond the supposed limits or environment set by the game.”_
+We’ve been using it to create better money and build decentralized financial systems, applications & organizations.
+
+-   Overall, for fairer, more decentralized & transparent societies
+-   Societies that make it hard to centralize power & take advantage of the masses
+-   Societies where individuals are empowered to act & make a difference rather than being told what to do as cogs in a machine that is beyond their understanding
+-   …
```

Comment:
Is this meant to be `...`?

Reply:
yeah, intentional because its about listing all the ways its praised for before taking it down to reality 😂